.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "extends": ["airbnb", "plugin:react/recommended"],
     "plugins": ["react"],
     "rules": {
+      "linebreak-style": 0,
```

Comment:
- [ ] Please don't edit the `eslintrc` file. These are the standard which needs to be maintained between all the `Micronauts` and make your code better.

Reply:
Hello Dipesh,

Thank you for your review. But I am a bit confused. I reached out to you on slack. Please check it